### ISSUE TITLE
Change to save the apt repos to openhab2.list

### ISSUE THREAD

**Issue by rkoshak** [openhab/openhab-docs#88]
Update the Linux installation instructions to save the apt- repos to /etc/apt/sources.list.d/openhab2.list instead of openhab.list. This will prevent overriding openhab.list for those who are migrating from OH 1 and used the apt-get install or confusing apt with left over OH 1 openhab.list.distUpgrade files.

**Comment by rkoshak** [comment]
Sorry, I'm still feeling my way around how GitHub works. Apparently when one submits a pull request it creates a new issue instead of giving you the chance to link it with this one. Or I missed that option.

**Comment by kaikreuzer** [comment]
Opening a PR does not create an issue, only a PR. You can then add a comment "fixes #xyz" to your PR and this will link it to the issue (and the issue is auto-closed once the PR is merged).

**Comment by rkoshak** [comment]
Thanks! I need to learn me some GitHub apparently.
